drools-wb-webapp/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -4,6 +4,12 @@
     <!-- These exclusions were added by Errai to avoid deploying client-side classes to the server -->
     <!-- End of Errai exclusions -->
 
+    <!-- These exclusions were added by Errai to avoid deploying client-side classes to the server -->
+    <!-- End of Errai exclusions -->
+
+    <!-- These exclusions were added by Errai to avoid deploying client-side classes to the server -->
+    <!-- End of Errai exclusions -->
+
```

Comment:
Please do not include these changes in PR.

Reply:
A reminder, we shouldn't these changes of `beans.xml` in this PR.

Reply:
@danielzhe please, ensure no changes are done in beans.xml as part of drools-wb#1214 PR